packages/cookies-tooltip/styles.tsx

```diff
@@ -0,0 +1,128 @@
+import styled from 'styled-components'
+
+const ExternalLink = styled.a.attrs({
+  target: '_blank',
+  rel: 'nofollow noopener',
```

Comment:
**not required:** maybe also use `noreferrer` ?

Reply:
So, I think we not need to use `noreferrer`, 'cause we needed to know where we went from to terms;